pontoon/sync/tasks.py

```diff
@@ -99,38 +98,74 @@ def sync_project(
 
     # Sync source strings. We cannot sync sources if locale is specified,
     # because that would apply any source string changes to the specified locale only.
+    if locale:
+        locales = db_project.locales.filter(pk=locale.pk)
+    else:
+        locales = db_project.locales.all()
+
+    if not locales:
+        log.info(
+            "Skipping syncing translations for project {0}, no locales to sync "
+            "found within.".format(db_project.slug)
```

Comment:
We're now skipping the entire sync (not just translations, but also source strings), so let's change this to:
```
log.info("Skipping project {0}, no locales to sync found.".format(db_project.slug))
```